src/js/components/molecules/tooltip-toggle/tooltip-toggle.ts

```diff
@@ -37,45 +40,44 @@ export class TooltipToggle extends Vue implements TooltipToggleProps {
         this.visible = true;
     }
 
-    toggle(): void {
+    showTooltip():void {
         if (!this.visible) {
             this.show();
         } else {
             this.hide();
         }
     }
 
-    hide():void {
+    hideTooltip(): void {
+        if (this.visible)  {
+            this.hide();
+        }
+    }
+
+    async hide():Promise<void> {
         this.visible = false;
     }
 
     tooltipEnter(el, done) {
         const tooltipElement = <HTMLElement> el;
-        const iconElement = <HTMLElement> this.$refs.button.$el;
-        const { height: tooltipHeight, width: tooltipWidth } = <DOMRect> tooltipElement.getBoundingClientRect();
-        const { x: iconX, width: iconWidth } = <DOMRect> iconElement.getBoundingClientRect();
+        const clickableELement = this.$refs.button;
 
-        const iconCenterOffset = iconWidth / 2;
-        let xAxisCorrection = tooltipWidth / -2 + iconCenterOffset;
+        // @ts-ignore
+        const { height: elementHeight } = <DOMRect> clickableELement.getBoundingClientRect();
 
-        if (iconX + xAxisCorrection < 0) {
-            xAxisCorrection = 0;
-            this.tooltipLeft = true;
-        } else {
-            this.tooltipLeft = false;
-        }
+        this.yAxisCorrection =  elementHeight + (this.buildingUnit / 2);
 
         TweenLite
             .fromTo(tooltipElement, this.duration,
-                    { autoAlpha: 0, x: xAxisCorrection, y: this.buildingUnit * -1 },
-                    { autoAlpha: 1, x: xAxisCorrection, y: 0, ease: Power2.easeOut })
+                    { autoAlpha: 0, x: 0, y: this.yAxisCorrection - this.buildingUnit },
+                    { autoAlpha: 1, x: 0, y: this.yAxisCorrection , ease: this.ease })
             .eventCallback('onComplete', done);
     }
 
     tooltipLeave(el, done) {
         const { height } = <DOMRect> el.getBoundingClientRect();
         TweenLite
-            .to(el, this.duration, { autoAlpha: 0, y: this.buildingUnit * -1, ease: Power2.easeIn })
+            .to(el, this.duration, { autoAlpha: 0, y: this.yAxisCorrection - this.buildingUnit, ease: this.ease })
```

Comment:
You should EaseOut when an element appears on the screen and EaseIn (accelerate) on the way out.